lib/uwb/include/uwb/protocols/fira/FiraDevice.hxx

```diff
@@ -190,11 +190,13 @@ enum class Channel {
 
 /**
  * @brief See FiRa Consortium UWB MAC Technical Requirements v1.3.0, Section
- * 7.5.3.3, Table 53.
+ * 7.5.3.3, Table 53; and FiRa Consortium UWB Command Interface Generic Technical
+ * Specification v1.1.0, Section 8.3, Table 29
  */
 enum class PrfMode {
     Bprf = 0,
     Hprf = 1,
+    HprfSpecialDataRate = 2,
```

Comment:
We already have the PRF define with rates in `PrfModeDetailed`, so is this actually needed here?

https://github.com/microsoft/nearobject-framework/blob/f916043c3f76aaa1afeb507af8775e9bbf22350b/lib/uwb/include/uwb/protocols/fira/FiraDevice.hxx#L450-L454

Reply:
Oh I missed that, I'll update my code!